sample_trial/darren_definition.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
I guess this file is redundant. right?

Reply:
yep was created for testing no need to merge but wanted to leave it in the branch for the reviewer to understand the approach.